bundles/redhat8.10/packages.txt.gotmpl

```diff
@@ -11,7 +11,7 @@ sysstat
 yum-utils
 yum-plugin-versionlock
 NetworkManager
-cloud-init
+cloud-init-23.1.1-10.el8
```

Comment:
Any impact on vSphere with this change? https://github.com/mesosphere/konvoy-image-builder/blob/11f483cbdec87867c0bd11acbd1a3166c5f51ed5/ansible/roles/providers/tasks/vmware.yml#L22

Reply:
good call out let me see.

Reply:
the builds are fine in kib: https://github.com/mesosphere/konvoy-image-builder/actions/runs/12020675705/job/33509809738#step:10:3847

 i dont see this being an issue for vsphere, but we would need to create a cluster with an ova to find out

Reply:
Ok so I built the release template here https://github.com/mesosphere/konvoy-image-builder/actions/runs/12021007230/job/33511178734

Being used here https://github.com/mesosphere/konvoy2/actions/runs/12021800408/job/33513150225#step:5:7595

and the tests are passing so i think we're good.